src/components/Note/note.css

```diff
@@ -0,0 +1,35 @@
+.note-container{
+  width: 280px;
+  min-height: 300px;
+  border-radius: 20px;
+  margin: 20px;
+  padding-top:10px ;
+  padding-left:10px ;
+  padding-right:10px ;
```

Comment:
Can use shorthand property here - `padding: 10px 10px 0px 10px;`

Reply:
fixed this